src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -1420,7 +1422,7 @@ private class OffsetAndRule {
             return localConverted;           
         }
 
-
+#if FEATURE_WIN32_REGISTRY
```

Comment:
This is the condition used throughout this file for any code that uses the Windows registry, and this function takes a Win32Native.RegistryTimeZoneInformation parameter.

I'll change the condition above to be FEATURE_WIN32_REGISTRY instead of !PLATFORM_UNIX.